app/src/main/res/layout/rolled_dices_list_fragment.xml

```diff
@@ -1,20 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Because this layout only has 1 View in it, and because that one View is a ViewGroup (parent of all Layouts) you can omit this parent layout. Having few nested Views and ViewGroups is a great way to optimize your apps performance. This mean the GPU can draw less on screen. And yes, even if its blank/empty the GPU draws it